roles/contiv_network/tasks/v2plugin_local_install.yml

```diff
@@ -0,0 +1,38 @@
+- name: Remove v2plugin rootfs directory
+  file: state=absent path=/var/lib/v2plugin/rootfs
+
+- name: Create v2plugin rootfs directory
+  file: state=directory path=/var/lib/v2plugin/rootfs
+
+- name: Unpack v2plugin from local archive
+  unarchive:
+    src: "/var/contiv_cache/{{ contiv_v2plugin_tar_filename }}"
+    dest: /var/lib/v2plugin/rootfs/
+
+- name: Copy config.json for plugin container
+  copy: src=/var/contiv_cache/config.json dest=/var/lib/v2plugin
```

Comment:
`rootfs` and `config.json` looks pretty ephemeral, can we make it in temp dir instead ?

Reply:
sure, will do that soon as the -1 gets cleared